### PR TITLE
Better Job Playtime Requirements

### PR DESCRIPTION
**Pull request by qwertyquerty**
You must now gain playtime in specific departments to play said department's head role.

This is good because rounds are ruined very quickly when the CMO is going around smacking people with medkits and the HoS is locked in permabrig by the warden 5 minutes in.

This one goes out to oshibka.

@crossedfall WE HAVE CONFIG CHANGES THAT DON'T AUTOMATICALLY MIRROR WOOP WOOP WOOP

## Changelog
:cl:
tweak: time requirements are now based on playtime and are departmental
/:cl:

### CODE REVIEW

**Comment by Oshibkka** [comment]
this pleases me

**Comment by qwertyquerty** [comment]
Moral of the story: never merge master into your branch when your local master is durr cringe

**Comment by jlsnow301** [comment]
Oh thank FUCK

**Comment by Bastian0930** [comment]
considering a captain had a SINGLE HOUR, I think this needs to be added.

File: config/config.txt [hunk 1]
```diff
@@ -57,31 +57,31 @@ MENTOR_LEGACY_SYSTEM
 #MENTORS_MOBNAME_ONLY
 
 ## Comment this out to stop locally connected clients from being given the almost full access !localhost! admin rank
-ENABLE_LOCALHOST_RANK
+#ENABLE_LOCALHOST_RANK
```
Comment (ike709) [inline config/config.txt]: Why? Commenting this out just makes testing a pain. When will anybody be connecting locally on the production server?

Comment (ondrej008) [reply config/config.txt]: Cringe moment.

Comment (ike709) [reply config/config.txt]: I mean I can understand disabling this locally on the production server just to be paranoid, but making me have to comment this back in by default is lame.

Comment (qwertyquerty) [reply config/config.txt]: Forgot to comment this back in, one sec

**Review by ike709 — APPROVED** [review]

**Review by ondrej008 — APPROVED** [review]
cringe moment was removing the localhost rank and removing playtime from engineers, other than that, seems good

**Comment by qwertyquerty** [comment]
Any new player should be able to play any job in any department except security. Everyone has their own style of play and they should be able to play their intended department from the start.

**Comment by jlsnow301** [comment]
Honestly I'd be okay with this putting that captain's one hour requirement on ALL jobs other than assistant so absolutely new players can't ruin xdepartment either

<1hr playtime isn't even enough to understand very basic mechanics

**Comment by qwertyquerty** [comment]
The problem we were facing was not new players playing stuff like engi and not knowing what to do. The problem was players joining once, playing assistant, then leaving for a week. A week later they come back and see they are eligible for CE, and decide to play that.

**Comment by qwertyquerty** [comment]
Also those requirements are in minutes, meaning that to play Captain, a player must have 6 hours of playtime.

**Comment by qwertyquerty** [comment]
**Non AFK Playtime

**Review by Crossedfall — APPROVED** [review]
Good by me. I can get the config changes applied post merge.

**Pull request merged**